documentation/modules/security/proc-installing-your-own-ca-certificates.adoc

```diff
@@ -11,7 +11,7 @@ This procedure describes how to install your own CA certificates and keys instea
 The Cluster Operator automatically generates and renews the following secrets:
 
 `_CLUSTER-NAME_-cluster-ca`:: The cluster secret that contains the private key for the cluster CA.
-`_CLUSTER-NAME_-cluster-ca-cert`:: The cluster secret that contains a cluster CA certificate. The certificate contains a public key to validate the identity of Kafka brokers.
+`_CLUSTER-NAME_-cluster-ca-cert`:: The cluster secret that contains a cluster CA certificate. The certificate contains a public key to validate the identity of Kafka brokers and ZooKeeper nodes.
```

Comment:
This is user facing docs => users do not connect to ZooKeeper ... so I think this is confusing. I think it should remain as it was.